main.tf

```
@@ -0,0 +1,169 @@
+data "aws_region" "current" {
+}
+
+data "template_cloudinit_config" "this" {
+
+  part {
+    content_type = "text/x-shellscript"
+    content      = local.configure_script
+  }
+
+  part {
+    content_type = "text/x-shellscript"
+    content      = var.asg_additional_user_data
+  }
+}
```

```suggestion
data "template_cloudinit_config" "this" {
  gzip = false

  part {
    content_type = "text/x-shellscript"
    content      = local.configure_script
  }

  part {
    content_type = "text/x-shellscript"
    content      = var.asg_additional_user_data
  }
}
```

---

I can't exactly tell what you're suggesting to change here.. is it just gzip? because that setting is incorrect for this use case (you can try if you like, it immediately breaks the implementation)

---

I thought `gzip = true` was the one that breaks everything?

---

if not adding in the gzip bit, make this change instead 
```suggestion
data "template_cloudinit_config" "this" {
  part {
    content_type = "text/x-shellscript"
    content      = local.configure_script
  }

  part {
    content_type = "text/x-shellscript"
    content      = var.asg_additional_user_data
  }
}
```

---

nah, gzip = true is the default.